.gitignore

```diff
@@ -1,4 +1,4 @@
-# /
```

Comment:
These changes should go in your local .gitignore file. You can add these entries to ~/.gitignore_global on you local machine and it will apply to all you repositories.